lib/peep_repository.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'peep'
+
+class PeepRepository
+   
+    def all
+        peeps = []
+      # Executes the SQL query:
+      sql='SELECT id, content, time, user_id FROM peeps;'
+      result_set = DatabaseConnection.exec_params(sql, [])
+      result_set.each do |record|
+        peep = Peep.new
+        peep.id = record['id'].to_i
+        peep.content = record['content']
+        peep.time = record['time']
+        peep.user_id = record['user_id']
+        peeps << peep
+      end
+      return peeps
+    end
```

Comment:
Would you mind adding enters between certain blocks of code (e.g. between line 9 and 10 above) and between methods (e.g. between line 19 and 20). I think your code will be a bit more readable, especially when files become longer and more complex.